From 2292efb41bacfafe0d484e35d375d91ccccbc5e8 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Tue, 10 Mar 2015 10:49:19 +0800
Subject: [PATCH 0521/1594] MLK-10389 ARM: dtsi: imx7d: add iram property to
 support internal SRAM

commit f1f9c1ba4ca6014dbc326d6b7fb305f21a48f2cb from
git://git.freescale.com/imx/linux-2.6-imx.git

By doing this, we can allow SDMA driver to allocate its memory from
iram when using i.MX7D SoC.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index e721393..79ff692 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -1136,6 +1136,7 @@
 					<&clks IMX7D_AHB_CHANNEL_ROOT_CLK>;
 				clock-names = "ipg", "ahb";
 				#dma-cells = <3>;
+				iram = <&ocram>;
 				fsl,sdma-ram-script-name = "imx/sdma/sdma-imx7d.bin";
 			};
 
-- 
1.7.5.4

